sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/JsonToRow.java

```diff
@@ -57,6 +57,10 @@
  *   <li>{@link Schema.TypeName#DOUBLE}
  *   <li>{@link Schema.TypeName#BOOLEAN}
  *   <li>{@link Schema.TypeName#STRING}
+ *   <li>{@link Schema.TypeName#DECIMAL}
+ *   <li>{@link Schema.TypeName#DATETIME}
+ *   <li>{@link Schema.TypeName#MAP} (keys must be STRING; values must be BYTE, INT16, INT32, INT64,
```

Comment:
I understand the code does (and json spec as well) does not explicitly restricts value type of a map (https://github.com/apache/beam/pull/34347/files#diff-f6962eeb7e5675e74f35dd7c94c4550bbd06f62cef7d3067e85324238db75525R597). Consider just phrases as "value must be a supported field type"